tools/packaging/kernel/configs/fragments/arm64/ptp.conf

```diff
@@ -3,5 +3,5 @@
 # The implementation of ptp_kvm on arm is one experimental feature,
 # you need to apply private patches to enable it on your host machine.
 # See https://github.com/kata-containers/packaging/pull/998 for detailed info.
-CONFIG_PTP_1588_CLOCK=y
-CONFIG_PTP_1588_CLOCK_KVM=y
+#CONFIG_PTP_1588_CLOCK=y
```

Comment:
please document why you are commenting out this

Reply:
ok